apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -146,19 +153,34 @@ export default class AnimationPickerBody extends React.Component {
     if (this.props.hideBackgrounds) {
       categories = categories.filter(category => category !== 'backgrounds');
     }
+    // Level-specific animations currently have the following categories:
+    // "level_animations", "backgrounds", "animals".
+    // We want to hide the "animals" category which has only one animation in it.
+    // It will be displayed in the "all" category added below.
+    // There is an 'animals' category because a level-specific animation was uploaded with the
+    // 'animals' category BEFORE the sprite upload feature dictated that level-specific animations
+    // can only be categorized as "costume" or "background".
```

Comment:
Can/should we fix this one animation so we don't need this extra logic?

Reply:
Answered above.